src/components/ContentNode/TabNavigator.vue

```diff
@@ -18,7 +18,7 @@
         {{ title }}
       </TabnavItem>
     </Tabnav>
-    <div class="tabs-content">
+    <div class="tabs-content" :key="currentTitle">
```

Comment:
Don't we want to maintain the state of a played video in a given tab, etc?

Also, I don't totally understand why the issue is only reproducible for videos—using something like text or images seems to work fine. Could you help explain the problem to me?

Reply:
Good questions. 

To maintain the video playback, we need to refactor this to use v-show, otherwise the v-if-ing will just destroy the video.

As for why it does this... I do not know. In devtools the components actually get the right props and everything, Vue just does not re-render the video component. Really weird honestly... Could be the internal diffing they use?